etc/init.d/Makefile.am

```diff
@@ -15,7 +15,7 @@ EXTRA_DIST = \
 	$(top_srcdir)/etc/init.d/zfs-zed.in \
 	$(top_srcdir)/etc/init.d/zfs.in
 
-$(init_SCRIPTS) $(initconf_SCRIPTS) $(initcommon_SCRIPTS):%:%.in
+$(init_SCRIPTS) $(initconf_SCRIPTS) $(initcommon_SCRIPTS):%:%.in Makefile
```

Comment:
I'm not sure I understand why this change is needed.

Reply:
As described in the commit message:
https://github.com/zfsonlinux/zfs/pull/9955/commits/77acb93b16f3ced9adfc77184275d395c4e7e26a

> This brings it in line with the example:
> https://www.gnu.org/software/automake/manual/html_node/Scripts.html
>
> This way, if the substitution code is changed, they should update.

It's just a correctness thing I noticed while debugging the actual issue. If you don't want it, it's not that big of a deal and I can strip that commit from the stack.

Reply:
Ahh, I see my mistake.  I should have looked at the commit message for each change and not the squashed PR.  That makes sense and I think we should keep it.